src/discolight/augmentations/imagecompression.py

```diff
@@ -0,0 +1,44 @@
+"""A Image compression augmentation."""
+import cv2
+from discolight.params.params import Params
+from .augmentation.types import ColorAugmentation, BoundedNumber
+from .decorators.accepts_probs import accepts_probs
+
+
+@accepts_probs
+class ImageCompression(ColorAugmentation):
+
+    """Image compress the given image.
+
+    Works for both jpeg/jpg and png format.
+    This function is a lossy JPEG compression operation.
+    """
+
+    def __init__(self, strength):
+        """Construct a Image compression augmentation.
+
+        You should probably use the augmentation factory or Discolight
+        library interface to construct augmentations. Only invoke
+        this constructor directly if you know what you are doing.
+        """
+        super().__init__()
+        self.strength = strength
+
+    @staticmethod
+    def params():
+        """Return a Params object describing constructor parameters."""
+        return Params().add(
+            "strength",
+            "Compression strength between 0 to 100",
```

Comment:
```suggestion
            "Compression strength",
```
Using the `BoundedNumber` type will cause information about the  bound to automatically appear in the documentation, so you don't need to add it to the param description.